jobs/pr_gate/docker/monorail/config.json

```diff
@@ -2,7 +2,7 @@
     "CIDRNet": "172.31.128.0/22",
     "amqp": "amqp://localhost",
     "apiServerAddress": "172.31.128.1",
-    "apiServerPort": 9090,
+    "apiServerPort": 9030,
```

Comment:
what exactly is the 'apiServerPort' used for.. I always thought it was same as northbound-api-router. :-)
So checking if the change is needed?